laliga/contracts/Golazos.cdc

```diff
@@ -934,59 +926,59 @@ pub contract Golazos: NonFungibleToken {
 
     /// An interface containing the Admin function that allows minting NFTs
     ///
-    pub resource interface NFTMinter {
+    access(all) resource interface NFTMinter {
```

Comment:
Should we remove all methods defined in this interface and add comments to make it clear that it doesn't allow defining access control anymore (kept in the contract only because removal is not a valid change for contract update)?

Also, should we create a `Operate` entitlement and use it for admin functions? It should not be necessary since the Admin resource is not meant to ever be exposed publicly, but the entitlement may improve readability still.

Reply:
removed the methods in the interface and added a comment.

Not adding the Operate entitlement because like you said the admin resource will not be linked publicly. Also i think it may be confusing it there is an entitlement for the mint method. It may leave the impression that the resource is accessible publicly which is not the case

Reply:
> It may leave the impression that the resource is accessible publicly which is not the case
without the entitlement, it leaves the impression that the resource is accessible publicly. but this is a blocker though. I just feel the entitlement helps readability

Reply:
it also relates to Josh's comment [here](https://github.com/dapperlabs/nba-smart-contracts/pull/248#discussion_r1541905999). Having an entitlement for the mint method but not others makes it seem like the Admin resource can be accessed in different ways or by different people

Reply:
> Having an entitlement for the mint method but not others makes it seem like the Admin resource can be accessed in different ways or by different people

yea i feel if we add the new entitlement it seems more readable to use it for all admin methods not just the mint method (we did that for Pinnacle and PackNFT/PDS when applicable)

Reply:
Yea, I can do that for all admin methods

Reply:
added `Mint` and `Operate` entitlements just like in the pinnacle contract